docs/subnets/deploy-a-gnosis-safe-on-your-evm.md

```diff
@@ -0,0 +1,810 @@
+Signature: 0x094e84aab062cb03f9abca3b80fb9931934c83920024fb8fa83b7b8d1a2aab305ab1f4d54e3a59ad7633f3f36d5db9b9976db268e05e0559c1c017fd3836540020
+```
+
+#### Submit
+```zsh
+yarn safe submit-proposal 0x2837eb329c41078c97e2450eabf0b73caae94d08db06a5d9fe2084d33ef3f4cc
+```
+
+Output:
+```zsh
+Running on subnet
+Using Safe at 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114 with 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
+Ethereum transaction hash: 0x99b35740246b91e5137f0128427e220ec7772aab17b20b6b9d4bcc7e0c73685f
+```
+
+Now that we've successfully submitted a proposal, lets check the owners of our safe by using the `info` task:
+
+```zsh
+yarn safe info --network subnet 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114
+```
+
+Output:
+```zsh
+Checking Safe at 0x1DE5B48F80eC78Bf74644EFdCbB5750Cb7B25114
+Singleton: 0x95CA0a568236fC7413Cd2b794A7da24422c2BBb6
+Version: 1.3.0
+Owners: 0x82DdaF3f1fcd3c18F5664cD7fb12bD8C38D5d4ba,0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
+Threshold: 2
+Nonce: 1
+Fallback Handler: 0x0000000000000000000000000000000000000000
+Modules: 
+```
+
+It is worth noting that you can also check the owners of the safe by using [Hardhat with your Custom EVM](https://docs.avax.network/dapps/smart-contracts/using-hardhat-with-the-avalanche-c-chain#interact-with-smart-contract).
+
+As shown above, `Owners` now includes a new address and `threshold`, the amount of signatures needed to execute a transaction, has increased to 2.
+
+### Send Native Currency from your Safe
+Lets apply the very same steps above to a workflow where we send the Native Currency of your Subnet to an EOA.
+This part of the tutorial requires that your Safe holds at least 1000 Native Tokens. You can send assets to your Safe the same way you would send Avax using [Metamask](https://metamask.zendesk.com/hc/en-us/articles/360015488931-How-to-send-tokens-from-your-MetaMask-wallet). To add your subnet to MetaMask, [please read this excerpt](../subnets/create-a-fuji-subnet#connect-with-metamask).
+
+
+Just as before, we will sign and submit the transaction hash. This example uses two signers due to an increased `threshold` from our previous Safe tx.
+
+To include our second signer, we will have to import another _private key_ into the project by adding it to `.env` and `hardhat.config.ts` in our `safe-tasks` project.
+
+Example `.env`:
+
+```js
+PK="56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027"
+PK2="cd30aef1af167238c627593537e162ecf5aad1d4ab4ea98ed2f96ad4e47006dc"
+INFURA_KEY=""
+# Used for custom network
+NODE_URL="http://127.0.0.1:49435/ext/bc/2Ek1MWR7jiEJr3o9tuJAH79JkuERzKqQDcR2s6R2e5Dyz54Wit/rpc"
+NETWORK="subnet"
+```
+
+Example `hardhat.config.ts`
+
+```ts
+// Load environment variables.
+dotenv.config();
+const {
+  NETWORK,
+  NODE_URL,
+  INFURA_KEY,
+  MNEMONIC,
+  PK,
+  PK2,
+  SOLIDITY_VERSION,
+  SOLIDITY_SETTINGS,
+} = process.env;
+
+...
+networks: {
+  subnet: {
+        url: NODE_URL,
```

Comment:
${NODE_URL}, does this need to be like this, as `${PK}`, `${PK2}` is used below.

Reply:
No but I am open to changing it